src/AccessibilityInsights.SharedUx/Controls/ScannerResultControl.xaml.cs

```diff
@@ -35,7 +36,16 @@ public partial class ScannerResultControl : UserControl
         /// <summary>
         /// Keeps track of if we should automatically set lv column widths
         /// </summary>
-        public bool HasUserResizedLvHeader { get; set; }
+        public bool HasUserResizedLvHeader 
```

Comment:
This could get a little stick when we add a second control for the framework issues. I'm not sure how we'll want to handle it.